metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/Workflow.java

```diff
@@ -77,4 +79,21 @@ public AbstractMetisPluginMetadata getPluginMetadata(PluginType pluginType) {
     }
     return null;
   }
+
+  public boolean isPluginTypeAfterBasedPluginType(PluginType basedPluginType,
+      PluginType pluginType) {
```

Comment:
Additionally, this method is called in the setting where we actually want to check if one of a given set of plugin types can be the base for a given plugin type. We should change this method so that it accepts a set of base plugin types so that this check becomes cheaper.